pkg/services/serviceaccounts/database/store.go

```diff
@@ -270,9 +270,6 @@ func (s *ServiceAccountsStoreImpl) SearchOrgServiceAccounts(ctx context.Context,
 	}
 
 	err := s.sqlStore.WithDbSession(ctx, func(dbSession *db.Session) error {
-		sess := dbSession.Table("org_user")
```

Comment:
I didn't actually remove that, it's just re-ordering "first count then fetch", to allow counting only.